pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -190,22 +185,13 @@ in {
     # incompatibleKernelVersion = "4.20";
 
     # this package should point to the latest release.
-    version = "2.0.0";
+    version = "2.0.1";
 
-    sha256 = "1kriz6pg8wj98izvjc60wp23lgcp4k3mzhpkgj74np73rzgy6v8r";
+    sha256 = "0wmw823ildwm9rcfyk22pvzg100yhps3y9hfjlrpspfd1hhkbp0d";
 
-    extraPatches = [ linux-rt-patch ];
+    extraPatches = [ ];
   };
 
-  zfsUnstable = common {
-    # comment/uncomment if breaking kernel versions are known
-    # incompatibleKernelVersion = "4.19";
-
-    # this package should point to a version / git revision compatible with the latest kernel release
-    version = "2.0.0";
-
-    sha256 = "1kriz6pg8wj98izvjc60wp23lgcp4k3mzhpkgj74np73rzgy6v8r";
-
-    extraPatches = [ linux-rt-patch ];
-  };
+  # this package should point to a version / git revision compatible with the latest kernel release
+  zfsUnstable = zfsStable;
```

Comment:
I actually would keep the old structure and just make it the same as `zfsStable`. We likely need this in future again.

Reply:
You could just copy pasta the structure of stable above.

Reply:
I agree with Mic92. If we add and remove this block each time a release candidate comes out, the diffs become quite messy and hard to understand.